### PR TITLE
Add Pixi locator

### PR DESCRIPTION
**Pull request by renan-r-santos**
Fixes https://github.com/microsoft/python-environment-tools/issues/46

I'd like to reopen the discussion about adding support for Pixi given the community interest after the above issue was closed and to bring feature parity with the `js` locator.
Because Pixi environments have a unique directory layout, the proposed solution is simple enough that it doesn't need any process spawning.

Sister PR:
- https://github.com/microsoft/vscode-python/pull/24442

### CODE REVIEW

**Comment by renan-r-santos** [comment]
@microsoft-github-policy-service agree

**Comment by renan-r-santos** [comment]
@karthiknadig I added a commit based on https://github.com/microsoft/python-environment-tools/pull/175 to fix the unrelated Homebrew CI issue. Even if `brew update` fails, running it somehow fixes the subsequent `brew install`. I also confirmed the issue only happens on Homebrew 4.4.6, released ~16 hours ago. Any Homebrew image <=4.4.5 works fine but I couldn't pinpoint the upstream commit causing the issue.
Let me know if this change is ok or if you prefer pinning the Homebrew image to something else.

**Comment by karthiknadig** [comment]
Thanks for the fix. What you have is good. May be a comment in the YML and docker file, since it looks odd that we are doing this. Also it will remind us to remove it in the future.

File: crates/pet/README.md [hunk 1]
```diff
@@ -66,53 +66,55 @@ It could have been created by other tools like `poetry`, `pipenv`, etc. Hence we
 
 4. Homebrew
    These are always stored in a custom (global) lcoation.
-5. Conda
+
+5. Pixi
```
Comment (DonJayamanne) [inline crates/pet/README.md:70]: Awesome,

**Review by karthiknadig — APPROVED** [review]

**Review by anthonykim1 — APPROVED** [review]

**Review by DonJayamanne — APPROVED** [review]

**Pull request merged**